numpy/lib/function_base.py

```diff
@@ -5403,12 +5403,12 @@ def insert(arr, obj, values, axis=None):
     return new
 
 
-def _append_dispatcher(arr, values, axis=None):
+def _append_dispatcher(arr, values, dtype=None, axis=None):
     return (arr, values)
 
 
 @array_function_dispatch(_append_dispatcher)
-def append(arr, values, axis=None):
+def append(arr, values, dtype=None, axis=None):
```

Comment:
The documentation for appended needs to be updated to explain what the dtype parameter is.

Reply:
I added the parameter to the function documentation.

Reply:
`dtype` must be the last parameter, to not break users who currently wrote: `np.append(arr, 1, 0)`